Automattic/wp-calypso
# Plans: Position CRM Free card alongside Jetpack Free

Relates to #56103.

#### Changes proposed in this Pull Request

* On the pricing page:
  * move the Jetpack CRM Free card out of the main product grid, and down onto the same row as Jetpack Free; and
  * update the gap between cards to a uniform 16px all the way around.
* On the Jetpack CRM Free card:
  * update the title from "CRM Free" to "Jetpack CRM Free";
  * update the description (see reference capture);
  * update the call-to-action button from "Get CRM" to "Start for free"; and
  * update the features list (see reference capture).
* On the Jetpack Free card:
  * update the description (see reference capture); and
  * update the features list (see reference capture).
* Add a `min-height` for free product descriptions: at least 2 lines of text, accounting for a line height of 1.5.

#### Testing instructions

**NOTE:** This PR should be tested in both Calypso Blue and Calypso Green.

1. Navigate to the Plans/Pricing page in your testing environment.
2. Verify the following:
  a. Jetpack CRM Free appears at the bottom of the pricing grid;
  b. on Calypso Green: Jetpack CRM Free shares a row with Jetpack Free, each taking up 50% of the page width;
  c. on Calypso Blue: Jetpack CRM Free takes up the full page width;
  d. Jetpack CRM Free has an updated title, description, call-to-action button, and features list compared to production;
  e. Jetpack Free has an updated description and features list compared to production; and
  f. as the screen width changes, cards stretch and squeeze responsively and intuitively.

#### Reference capture

https://user-images.githubusercontent.com/670067/136219488-828bfb4d-1ecf-4daa-b002-c067c6af8cc9.mp4
